docker-compose.yml

```
@@ -0,0 +1,55 @@
+    environment:
+      - NODE_ENV=production
+      - PORT=9999
+      - HOSTNAME=0.0.0.0
+    restart: unless-stopped
+
+volumes:
+  sql_server_data:
+
+networks:
+  backend:
+    driver: bridge
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)